requirements.txt

```diff
@@ -3,3 +3,4 @@ psycopg2-binary
 pytz
 six
 typing
+django-pg-returning
```

Comment:
I don't include this library to `requirements.txt` file as it is optional. To my mind it is better to create `requirements-test.txt` file for testing purposes and install in `.travis.yml` or other CI configuration